media-libs/libjxl/libjxl-0.9.1.ebuild

```diff
@@ -101,10 +101,6 @@ multilib_src_install() {
 	find "${ED}" -name '*.a' -delete || die
 }
 
-pkg_preinst() {
```

Comment:
so we drop the call to `gnome2_gdk_pixbuf_savelist`, but now after https://github.com/gentoo/gentoo/pull/35008/files#r1472828661, we won't be running it at all unless people call gnome2_gdk_pixbuf_update directly instead of gnome2_pkg_postinst...

Reply:
No, if people call `gnome2_pkg_preinst`, then `gnome2_pkg_postinst` will work just fine.